examples/dev/deployment_resource.tf

```diff
@@ -0,0 +1,152 @@
+terraform {
+  required_providers {
+    grid = {
+      source = "threefoldtech/grid"
+    }
+  }
+}
+
+provider "grid" {
+}
+
+resource "grid_capacity_reserver" "cap1" {
+  farm   = 1
+  cpu    = 2
+  memory = 2048
```

Comment:
both deployments use `cap1` and they require 4096 memory so not enough for both maybe use `cap2` for second deployment?